Marketplace contribution article (#284)
demisto/content-docs: docs/contributing/marketplace.md

1. On Cortex XSOAR, contribute as described in [this article](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/marketplace/content-pack-contributions.html).

2. You will then be promoted with a toast saying that your contribution is ready for review, with a link to a form you need to fill to complete your contribution.
In addition, you will receive an email with the link to the form.

3. In the form you will be asked for the following:
     - Fill in additional details, such as: pack name, author and description.
     - Log in to your GitHub account, so you can later participate in the review process as part of the pull request that will be opened.
     - Sign the [Palo Alto Networks Contributor License Agreement](https://github.com/demisto/content/blob/master/docs/cla.pdf).
     
4. After submitting the form, you will be redirected to a page in which it will say that pack was accepted and is being processed.  
5. A GitHub branch will be created in the [xsoar-contrib Content repository fork](https://github.com/xsoar-contrib/content), based on the pull request will be opened.

> 6. An invitation to join the xsoar-contrib organization will be sent to you so xsoar-bot will be able you to a GitHub team to give you write permissions to the branch created.

Reviewer: ```suggestion
6. An invitation to join the **xsoar-contrib** organization will be sent to you. You will be allowed to update only your pull request.
```

7. The pull request will then be created, and a reviewer will be assigned to it.
8. You can now modify the files changed in the pull request as part of the review.

>  

Reviewer: add note here , in case they want to do updates in the pack ,the updates must be through github or close the pr and re-contribute again

---
id: marketplace

> title: Contributing From Cortex XSOAR Marketplace

Reviewer 2: I don't think we need `Marketplace` in the title. The process is actually contributing from Cortex XSOAR to the Marketplace.
```suggestion
title: Contributing From Cortex XSOAR
```

---

Cortex XSOAR v6.0 introduces a Marketplace which is the central location for installing, exchanging, contributing, and managing all of your content, including playbooks, integrations, automations, fields, layouts, and more. For more information, you can read all about the [Marketplace](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/marketplace/marketplace-overview.html).

> This article walks you through the process of contributing content from the Marketplace.

Reviewer 2: ```suggestion
This article walks you through the process of contributing content from the Cortex XSOAR Server to the Marketplace. This flow is meant to ease and speed up the contribution process for individual contributors who are not fully familiar with GitHub and how the Pull Request process works. Technology Partners should not use this flow and submit their Packs via a [GitHub Pull Request](checklist#pull-request-checklist).  
```
